filebeat/plan.sh

```diff
@@ -0,0 +1,40 @@
+pkg_name=filebeat
+pkg_origin=core
+pkg_version="6.2.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_build_deps=(core/go core/git core/make core/gcc)
+pkg_bin_dirs=(bin)
+pkg_binds_optional=(
+  [kibana]="port"
+  [elasticsearch]="http-port"
+  [logstash]="port"
+)
+pkg_svc_user="hab"
```

Comment:
I would remove this since it's the default

Reply:
@elliott-davis It is done!